SETUP/upgrade/12/20180216_alter_access_log.php

```diff
@@ -0,0 +1,23 @@
+<?php
+$relPath='../../../pinc/';
+include_once($relPath.'base.inc');
+
+header('Content-type: text/plain');
+
+// ------------------------------------------------------------
+
+echo "Increasing size of access_log.activity column..\n";
+$sql = "
+    ALTER TABLE access_log
+        CHANGE COLUMN activity activity VARCHAR(32) NOT NULL DEFAULT '';
```

Comment:
If you replace "CHANGE" with "MODIFY", you only need to give the column name once.

Reply:
Neat -- why haven't I noticed that before?